cli/src/cmd/forge/build/core.rs

```diff
@@ -52,6 +52,16 @@ pub struct CoreBuildArgs {
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub ignored_error_codes: Vec<u64>,
 
+    #[clap(
+        help_heading = "COMPILER OPTIONS",
+        help = "The minimum severity level that is treated as a compiler error",
+        long,
+        value_name = "SEVERITY",
+        default_value_t=Severity::Error.to_string()
+    )]
+    #[serde(skip)]
+    pub compiler_severity_filter: String,
```

Comment:
since error is the default, this is essentially like a `--deny-warnings` flag which would treat warnings as errors as well, right?

imo this is only what we need?

Reply:
i agree, made a new push to reflect that sentiment now, since I will be unavailable rest of day.